### PR TITLE
Using a normal bundle instead of a factored bundle for phishing-detect (build-sys)

### LINKED ISSUE #11971: "Continue at your own risk" button does not work
Dear manager,

Metamask protection appears when opening some sites. I was taking the risk. I can't access the site even though I took this risk for a week. What should I do. I am waiting for your help, please

Metamask

### PR DESCRIPTION
**Pull request by ryanml**
Fixes: https://github.com/MetaMask/metamask-extension/issues/11971 - since this was previously a factored build, `phishing.html` was not referencing the correct bundled js file: https://github.com/MetaMask/metamask-extension/blob/develop/app/phishing.html#L8 - this should be fine as a normal bundle

### CODE REVIEW

**Review by kumavis — DISMISSED** [review]

**Comment by kumavis** [comment]
> Error: SourcemapValidator - failed to load source file for "phishing-detect-0.js"
sourcemap validator needs to be updated

**Comment by ryanml** [comment]
@kumavis - validator updated 👍

**Review by rekmarks — DISMISSED** [review]
LGTM!

File: development/sourcemap-validator.js [hunk 1]
```diff
@@ -24,7 +24,6 @@ async function start() {
     `common-0.js`,
     `background-0.js`,
     `ui-0.js`,
-    'phishing-detect-0.js',
```
Comment (rekmarks) [inline development/sourcemap-validator.js:27]: I approved, but we don't want this in here?

Comment (rekmarks) [reply development/sourcemap-validator.js:27]: By our build system lingo, it's no longer a "factored build", so no, we don't.

**Comment by kumavis** [comment]
we should still validate it

**Review by kumavis — APPROVED** [review]

**Pull request merged**